README.md

```diff
@@ -36,9 +36,7 @@ We provide packaging for various distributions, but here's a quick method to
 build from source.
 
 ```
-git clone https://github.com/helix-editor/helix
-cd helix
-cargo install --path helix-term
+cargo install helix-term --git https://github.com/helix-editor/helix
 hx --grammar fetch
 hx --grammar build
```

Comment:
@the-mikedavis do we need these two?

Reply:
oh good point, those can go because this is done automatically by the helix-term build phase 👍 

```suggestion
```

Reply:
If you do it this way/via `build.rs`, where do the `.so` grammars end up in? I imagine it could be in some temporary cargo build location that's then wiped. The user still needs to clone so they can get a copy of `runtime/`

Reply:
Oh yeah that would be problematic. Even if the .so files end up in the runtime directory, you'd need to link/copy the queries and themes

Reply:
It seems that the grammars end up in the temporary checkout directory (`$HOME/.cargo/git/checkouts/helix-e461703849afe78e/3e78b8f/runtime/grammars` in my case), so these two commands should still be necessary.